src/assets/css/style.css

```diff
@@ -0,0 +1,51 @@
+/* NAVBAR */
+
+.navbar {
+  -webkit-user-select: none;
```

Comment:
Could we move `-webkit-user-select` from the navbar to the whole window? This will make the app feel more native since native applications don't usually allow text selection.